packages/plugin-data/src/batcher.ts

```diff
@@ -55,10 +55,10 @@ export class Batcher {
     jobId: string,
     doneCallback?: (...args: [{ job: JobInfo }]) => void
   ): Promise<JobInfo> {
-    const job: jsforceJob = this.conn.bulk.job(jobId);
+    const job: Job = this.conn.bulk.job(jobId);
     const jobInfo: JobInfo = await job.check();
 
-    this.bulkBatchStatus(jobInfo);
+    this.bulkStatus(jobInfo, undefined, undefined, true);
```

Comment:
JSYK: A way to avoid passing undefined is to change the method signature to take a config object (best way) or possibly to use default values.